firebase/src/components/surgeryPage/surgeryPagePublicInfo.tsx

```diff
@@ -4,53 +4,147 @@ import {
                     value={surgeryNameDraft}
                     onChange={handleNameChange}
                 />
             </Grid>
+            <Grid item xs={12} md={6} container direction="column" alignItems="stretch">
+                <TextField
+                    label="Háziorvosi rendelő neve"
+                    variant="outlined"
+                    error={isFieldValueEmpty(surgerySurgeryNameDraft)}
+                    helperText={isFieldValueEmpty(surgerySurgeryNameDraft) ? "A rendelő neve nem lehet üres" : "Itt megadhatja a rendelő nevét"}
+                    value={surgerySurgeryNameDraft}
+                    onChange={handleSurgeryNameChange}
+                />
+            </Grid>
+            <Grid item xs={12} md={6} container direction="column" alignItems="stretch">
+                <TextField
+                    label="Irányítószám"
+                    variant="outlined"
+                    error={isFieldValueEmpty(surgeryPoBoxDraft)}
+                    helperText={isFieldValueEmpty(surgeryPoBoxDraft) ? "Az irányítószám nem lehet üres" : "Itt megadhatja a háziorvos nevét"}
+                    value={surgeryPoBoxDraft}
+                    onChange={handlePoBoxChange}
+                />
+            </Grid>
+            <Grid item xs={12} md={6} container direction="column" alignItems="stretch">
+                <TextField
+                    label="Város"
+                    variant="outlined"
+                    error={isFieldValueEmpty(surgeryCityDraft)}
+                    helperText={isFieldValueEmpty(surgeryCityDraft) ? "A város neve nem lehet üres" : "Itt megadhatja a háziorvos nevét"}
+                    value={surgeryCityDraft}
+                    onChange={handleCityChange}
+                />
+            </Grid>
+            <Grid item xs={12} md={6} container direction="column" alignItems="stretch">
+                <TextField
+                    label="E-mail cím"
+                    variant="outlined"
+                    error={isFieldValueEmpty(surgeryEmailDraft)}
+                    helperText={isFieldValueEmpty(surgeryEmailDraft) ? "A háziorvos emailcíme neve nem lehet üres" : "Itt megadhatja a háziorvos emailcímét"}
+                    value={surgeryEmailDraft}
+                    onChange={handleEmailChange}
+                />
+            </Grid>
+            <Grid item xs={12} md={6} container direction="column" alignItems="stretch">
+                <TextField
+                    label="Telefonszám"
+                    variant="outlined"
+                    error={isFieldValueEmpty(surgeryPhoneDraft)}
+                    helperText={isFieldValueEmpty(surgeryPhoneDraft) ? "A háziorvos telefonszáma neve nem lehet üres" : "Itt megadhatja a háziorvos telefonszámát pl.: +3612345678"}
+                    value={surgeryPhoneDraft}
+                    onChange={handlePhoneChange}
+                />
+            </Grid>
```

Comment:
Szerintem a telefonszám és az emailcím se legyen kötelező, az utóbbi egyrészt nem biztos, hogy van, az előbbit pedig nem biztos, hogy itt is listázni akarják.

Reply:
Javítom.